testing/InstallTestbed.ps1

```diff
@@ -0,0 +1,309 @@
+param (
+    [Parameter(Mandatory = $true)]
+    [string]$ResourceGroup,
+
+    [string]$VMUsername = "admin.ackbar",
+    [string]$VMName = "DC1",
+    [string]$LinuxVMName = "LS1",
+    [int]$NumberOfClients = 2
+)
+
+# If you were to need the password from the SetupTestbed.ps1 script, you could use this:
+# $Password = Get-Content "password.txt"
```

Comment:
Since you changed SetupTestbed to include the resource group name in the file name (I love that change by the way), I'd recommend you indicate that here in the comment.

Something like this:
```suggestion
# $Password = Get-Content "[resource group name].password.txt"
```